assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -65,10 +66,15 @@ const Hint = styled.div`
   background-color: ${({ theme }) => theme.colors.bg.v1};
 `;
 
+function safeZoneSelector({ state: { showSafeZone } }) {
+  return { showSafeZone };
+}
+
 function FramesLayer() {
   const { currentPage } = useStory((state) => ({
     currentPage: state.state.currentPage,
   }));
+  const { showSafeZone } = useCanvas(safeZoneSelector);
```

Comment:
Why not use the same selector syntax as the rest of the codebase?

const { showSafeZone } = useCanvas(({ state: { showSafeZone }}) => ({ showSafeZone }));

Reply:
Yep, @dmmulroy, we need to fix this. Could you pls send a followup pull request?

Reply:
see #2710